src/libstd/sys/windows/printing/msvc.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use sys_common::backtrace::*;
+use ffi::CStr;
+use str;
+use dynamic_lib::DynamicLibrary;
+use super::SymFromAddrFn;
+use io::prelude::*;
+
+pub fn print(w: &mut Write, i: isize, addr: u64, lib: &DynamicLibrary) -> io::Result<()> {
```

Comment:
Could this `lib` argument be called `dbghelp`? Just to emphasize which DLL it is